lib/ios/RNNComponentPresenter.m

```diff
@@ -127,15 +127,15 @@ - (void)mergeOptions:(RNNNavigationOptions *)options resolvedOptions:(RNNNavigat
 	if (options.topBar.largeTitle.visible.hasValue) {
 		[viewController setTopBarPrefersLargeTitle:options.topBar.largeTitle.visible.get];
 	}
-
+    
+    if (options.layout.componentBackgroundColor.hasValue) {
+        [viewController setBackgroundColor:options.layout.componentBackgroundColor.get];
```

Comment:
Took me ~1hour after upgrading why my overlays arent transparent. It was espetially missleading as typescript file says

```
/**
     * Set background color only for components, helps reduce overdraw if background color is set in default options.
     * #### (Android specific)
     */
    componentBackgroundColor?: Color;
```

So I was ignoring it. Does it mean `backgroundColor` is now useless in component layout?

Reply:
Hey @pie6k, you're pretty much on point. As long as you use component layout on its own (without being wrapped in stack/bottomTabs/...), only componentBackgroundColor takes effect. But if you use `stack` for example, `backgroundColor` still changes the root UINavigationController background color, so this affects things like topBar (if you set its color to 'transparent', then `backgroundColor` will show through).
Hopefully this clears things up :)